frontends/web/src/routes/settings/settings.tsx

```diff
@@ -195,6 +208,29 @@ class Settings extends Component<Props, State> {
                           <FiatSelection />
                         </div>
                         <div className="column column-2-3">
+                          <div>
+                            <h3 className="subTitle">{t('settings.info.title')}</h3>
+                            <div className="box slim divide m-bottom-large">
+                              { version !== undefined && update !== undefined ? (
+                                <SettingsItem
+                                  optionalText={version}
+                                  primaryText={t('settings.info.version')}
+                                  clickUrl={ update ? updatePath : null }>
```

Comment:
Could you try with `SettingsButton` component? It should support `onClick`, but maybe it needs a change to hide the icon

Reply:
Done :+1: